src/pages/landing/forex-trading/components/_hero.tsx

```diff
@@ -87,9 +82,29 @@ const TryButton = styled(LinkButton)`
     }
 `
 
-const HeroComponent = ({ title, content, background_data }: HeroProps) => {
+const Hero = ({ title, content }: HeroProps) => {
+    const [is_mobile] = useBrowserResize()
+
     return (
-        <BackgroundWrapper data={background_data}>
+        <BackgroundImageWrapper>
+            <StaticImageWrapper>
+                {is_mobile ? (
+                    <StaticImage
+                        src="../../../../images/common/landing/trade-fx-m.jpg"
+                        alt="forex trading"
+                        objectFit="cover"
+                        objectPosition="bottom right"
```

Comment:
Same here and other places. If the image is at the top of the page, add `loading="eager"` so that it loads faster

Reply:
done